### ISSUE TITLE
externe temperatursenoren nutzen

### ISSUE THREAD

**Issue by tlundi60** [mskg/homey-heating#170]
Hello, I've been using the app for a short time and I think it's very well done. one question, is it possible to use the temperature from external sensors, e.g. netatmo, for the control?
maybe there is a simple way or can only the actual temperature of the thermostats be evaluated
I currently have fritz dect thermostats 

regards tom

**Comment by Dj-spooky** [comment]
do you have any answer yet? I'm having the same problem

**Comment by tlundi60** [comment]
> do you have any answer yet? I'm having the same problem

Sorry no Solution vor this question 
Hope the programmer will do it in future

**Comment by mskg** [comment]
Hi Gents. The application tells other devices to reach a certain temperature at a specific point in time. For this purpose, an external measurement of the temperature is not really relevant. The device itself is responsible for reaching the temperature that it's set to. The application does not try to reach a specific temperature in a room by regulating a separate thermostat up and down.

Did I understand the question right?

**Comment by tlundi60** [comment]
Yes, that right but sometime ist is better to use the Temperaturen Fromm extern 
So you can geht the real room temp. And the System regulate this.

**Comment by mskg** [comment]
Understood. So basically compensate until the external probe reaches its setting.
